pkg/sysctl/sysctl.go

```diff
@@ -54,7 +60,31 @@ func parameterPath(name string) (string, error) {
 			return "", ErrInvalidSysctlParameter(name)
 		}
 	}
-	return filepath.Join(append([]string{prefixDir}, elems...)...), nil
+	return filepath.Join(append([]string{GetProcfs(), "sys"}, elems...)...), nil
+}
+
+// SetProcfs sets path for the root's /proc. Setting this value with a different
+// default value and executing it after GetProcfs causes panic.
+func SetProcfs(path string) {
+	if path == procFsDefault {
+		return
+	}
```

Comment:
Can you elaborate the rational behind this codepath? I find it confusing that `SetProcfs(procFsDefault)` is consistently a no-op:
```golang
SetProcfs("/foo")
SetProcfs(procFsDefault)
p := GetProcfs() // here p == "/foo"
SetProcfs(procFsDefault) // no panic
SetProcfs("/foo") // panic although could be idem-potent
```

Reply:
Yeah I don't see any rational behind it, this code was written over 1y ago. Removed.